types/lib/utils/settings.d.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   lib/utils/settings.html
+ */
+
+/// <reference path="boot.d.ts" />
+/// <reference path="resolve-url.d.ts" />
+
+declare namespace Polymer {
+
+  /**
+   * Legacy settings.
+   */
+  namespace settings {
```

Comment:
We should probably add the declarations for the following:

```ts
namespace settings {
  useNativeCSSProperties: boolean;
  useNativeCustomElements: boolean;
  useShadow: boolean;
}
```

Reply:
I moved the `@namespace` annotation from `settings` to `Settings` so at least that's correct now. But actually there is no way in Analyzer to model this object and its interface. Going to punt on the full interface for now. The main way people use this is with a `<script>` tag in their `index.html` anyway, so it's unlikely it would need to be references from TypeScript.